packages/code-connect/components/Modal/AlertModal.figma.tsx

```diff
@@ -0,0 +1,21 @@
+import figma from '@figma/code-connect';
+import { Modal } from '@patternfly/react-core';
+
+figma.connect(
+  Modal,
+  'https://www.figma.com/design/aEBBvq0J3EPXxHvv6WgDx9/PatternFly-6--Components-Test?node-id=2940-18403&t=IzSunfrnw18ti37Y-11',
+  {
+    props: {
+      size: figma.enum('Size', {
+        Small: 'small',
+        Medium: 'medium',
+        Large: 'large'
+      }),
+      children: figma.children('*')
+    },
+    example: (props) => (
+      // Documentation for Modal can be found at https://www.patternfly.org/components/modal
+      <Modal variant={props.size}>{props.children}</Modal>
```

Comment:
we don't have alert modals?

we have a title icon which lets them specify a status to add the correct icon to the header?
https://www.patternfly.org/components/modal#title-icon